geode-assembly/src/integrationTest/java/org/apache/geode/GeodeDependencyJarIntegrationTest.java

```diff
@@ -0,0 +1,96 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package org.apache.geode;
+
+import static org.junit.Assert.assertTrue;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Arrays;
+import java.util.Set;
+import java.util.TreeSet;
+import java.util.jar.JarFile;
+import java.util.jar.Manifest;
+import java.util.stream.Collectors;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+
+import org.apache.geode.test.junit.categories.RestAPITest;
+import org.apache.geode.util.test.TestUtil;
+
+@Category({RestAPITest.class})
+public class GeodeDependencyJarIntegrationTest {
+
+  private static final String GEODE_HOME = System.getenv("GEODE_HOME");
+  private Set<String> expectedClassPath;
+
+  @Before
+  public void loadExpectedClassPath() throws IOException {
+    String assemblyContent =
+        TestUtil.getResourcePath(AssemblyContentsIntegrationTest.class,
+            "/dependency_classpath.txt");
+
+    expectedClassPath = Files.lines(Paths.get(assemblyContent)).collect(Collectors.toSet());
+  }
+
+  @Test
+  public void verifyManifestClassPath() throws IOException {
+    Set<String> currentAssemblyContent = getManifestClassPath();
```

Comment:
Let's cleanup the variable names in here to be meaningful. Looks like a copy and paste another test. ;)